source/common/http/conn_manager_impl.h

```diff
@@ -257,6 +269,9 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
     void decodeTrailers(ActiveStreamDecoderFilter* filter, HeaderMap& trailers);
     void maybeEndDecode(bool end_stream);
     void addEncodedData(ActiveStreamEncoderFilter& filter, Buffer::Instance& data, bool streaming);
+    void sendLocalReply(ActiveStreamEncoderFilter* filter, bool is_grpc_request, Code code,
```

Comment:
I think the first param here is always set to nullptr? What is the intention to eventually allow proper restart for filters downstream? For now, I would just remove the parameter and add TODOs around allowing proper restart if the filter is also an encoding filter?

Reply:
Right, but as it is still a TODO, I'll remove it and add the comments to the `encodeHeaders()` and `encodeData()` calls that take the `nullptr` filter pointer as the first parameter in the implementation of `sendLocalReply()`.